src/CESMII.OpcUa.NodeSetImporter/UANodeSetCacheManager.cs

```diff
@@ -184,6 +184,15 @@ static List<ModelValue> OrderByDependencies(List<ModelValue> models)
         {
             var remainingModels = new List<ModelValue>(models);
             var orderedModels = new List<ModelValue>();
+            ///TODO-Markus: Next lines were removed in the CESMII original version but existed in mine. Not sure if they are needed, Can you check
+            //var standalone = models.Where(m => !m.Dependencies.Any()).ToList();
+            //orderedModels.AddRange(standalone);
+            //foreach (var imr in standalone)
+            //{
+            //    remainingModels.Remove(imr);
+            //}
+
+
 
```

Comment:
The first run through the do/while loop catches all those zero-dependency models already, so priming it was redundant code.